.travis.yml

```diff
@@ -3,11 +3,11 @@ sudo: false
 
 language: python
 
-cache:
-  apt: true
-  directories:
-  - $HOME/.cache/pip
-  - $HOME/.ccache
+# cache:
```

Comment:
@maxcopeland61 Question: Why do we need to modify .travis.yml in a DOC PR?

Reply:
Ugh. That's pretty bad of me... Not checking that there were still only two files modified. Have you reverted?

Reply:
@jnothman Reverted in https://github.com/scikit-learn/scikit-learn/commit/a4dcd70e84176a255654586bd20e78779191447f

Reply:
Thanks @qinhanmin2014 :/